prerequisite_settings.json

```diff
@@ -16,5 +16,9 @@
     },
     "python": {
         "version": "3.9"
+    },
+    "dapr":{
+        "version": "1.8.0",
+        "runtime": "1.8.1"
```

Comment:
Please change runtime version in title of PR request, to have consistency

Reply:
done